.github/workflows/playground_selenium.yaml

```diff
@@ -2,38 +2,77 @@ name: Playground Selenium Tests
 
 on:
   workflow_dispatch:
+    inputs:
+      branch:
+        description: "Select which Git branch to run against"
```

Comment:
Lets unify our messages and use the same message like grid nigtly
"Use workflow from"